pims/pyproject.toml

```
@@ -9,7 +9,7 @@ license = "Apache-2.0"
 dependencies = [
   "aiofiles==23.1.0",
   "celery==5.2.7",
-  "cytomine-python-client==3.0.0",
+  "cytomine-python-client @ file:///${PROJECT_ROOT}/../clients/python",
```

```suggestion
  "cytomine-python-client",
```

Normally, it is possible to specify it in `[tool.uv.sources]` and no need for `@ file:///${PROJECT_ROOT}/../clients/python"`

Although it is correct but your solution is more `pip` style and my suggestion is more targeted for the `uv` package manager